docs/providers/aws/events/http-api.md

```diff
@@ -271,3 +271,15 @@ provider:
   httpApi:
     payload: '1.0'
 ```
+
+### Detailed Metrics
+
+With HTTP API we may configure detailed metrics that can be used setup monitoring and alerting in Cloudwatch.
+
+Detailed Metrics can be turned on with:
+
+```yaml
+provider:
+  httpApi:
+    detailedMetrics: true
```

Comment:
Let's simply name it `metrics` (as it's on `apiGateway`)

Reply:
done!